plugins/rapid7_insightidr/help.md

```diff
@@ -40,6 +47,411 @@ Example input:
 
 ### Actions
 
+#### List Alerts for Investigation
+
+This action is used to retrieve a page of alerts associated with the specified investigation.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|id|string|None|True|The ID or RRN of the investigation|None|rrn:investigation:example:11111111-1111-1111-1111-111111111111:investigation:11111111|
+|index|integer|0|True|The optional zero-based index of the page to retrieve. Must be an integer greater than or equal to 0|None|1|
+|size|integer|100|True|The optional size of the page to retrieve. Must be an integer greater than 0 or less than or equal to 100. Default value is 100|None|100|
+
+Example input:
+
+```
+{
+  "id": "rrn:investigation:example:11111111-1111-1111-1111-111111111111:investigation:11111111",
+  "index": 1,
+  "size": 100
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|alerts|[]alert|True|A list of alerts associated with the investigation|
```

Comment:
Why the output variables are marked required.

Reply:
If plugin goes fine - all the requests pass - it means the key `alerts` have to be in the `output` object. Even if the `alerts` key is empty - still it will be in the `output`